sdk/communication/azure-communication-callautomation/azure/communication/callautomation/_shared/models.py

```diff
@@ -62,6 +64,9 @@ class CommunicationIdentifier(Protocol):
 BOT_DOD_CLOUD_GLOBAL_PREFIX = "28:dod-global:"
 BOT_GCCH_CLOUD_PREFIX = "28:gcch:"
 BOT_GCCH_CLOUD_GLOBAL_PREFIX = "28:gcch-global:"
```

Comment:
pls remove all bot MRI prefixes those won't be supported

Reply:
Removed.